management-core-resources/src/main/java/org/terracotta/management/embedded/StandaloneServer.java

```diff
@@ -97,17 +95,18 @@ public void start() throws Exception {
       HttpConfiguration httpConfig = new HttpConfiguration();
       httpConfig.setSecureScheme("https");
       httpConfig.setSecurePort(port);
-      httpConfig.setBlockingTimeout(0); // make it same as idleTime (defaults to 30s.)
```

Comment:
What changed here?

Reply:
That API was removed and replaced with HttpConfig.min{Request,Response}DataRate, which default to 0.